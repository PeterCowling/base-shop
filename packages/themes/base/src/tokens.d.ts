/**
 * Central colour-/spacing-/font-token catalogue.
 * Every token has a `light` value and may provide a `dark` variant.
 */
export interface Token {
    /** CSS colour / length / font list used in light mode */
    readonly light: string;
    /** Optional counterpart for dark mode */
    readonly dark?: string;
}
/** `--token-name`: `{ light, dark? }` */
export type TokenMap = Record<`--${string}`, Token>;
export declare const tokens: {
    readonly "--color-bg": {
        readonly light: "0 0% 100%";
        readonly dark: "0 0% 4%";
    };
    readonly "--color-fg": {
        readonly light: "0 0% 10%";
        readonly dark: "0 0% 93%";
    };
    readonly "--color-primary": {
        readonly light: "220 90% 56%";
        readonly dark: "220 90% 66%";
    };
    readonly "--color-primary-fg": {
        readonly light: "0 0% 100%";
        readonly dark: "0 0% 10%";
    };
    readonly "--color-accent": {
        readonly light: "260 83% 70%";
        readonly dark: "260 83% 70%";
    };
    readonly "--color-accent-fg": {
        readonly light: "0 0% 10%";
        readonly dark: "0 0% 10%";
    };
    readonly "--color-danger": {
        readonly light: "0 86% 97%";
        readonly dark: "0 63% 31%";
    };
    readonly "--color-danger-fg": {
        readonly light: "0 74% 42%";
        readonly dark: "0 93% 94%";
    };
    readonly "--color-success": {
        readonly light: "142 76% 97%";
        readonly dark: "142 72% 27%";
    };
    readonly "--color-success-fg": {
        readonly light: "142 72% 30%";
        readonly dark: "142 70% 94%";
    };
    readonly "--color-warning": {
        readonly light: "40 90% 96%";
        readonly dark: "35 90% 30%";
    };
    readonly "--color-warning-fg": {
        readonly light: "25 85% 31%";
        readonly dark: "40 90% 96%";
    };
    readonly "--color-info": {
        readonly light: "210 90% 96%";
        readonly dark: "210 90% 35%";
    };
    readonly "--color-info-fg": {
        readonly light: "210 90% 35%";
        readonly dark: "210 90% 96%";
    };
    readonly "--color-muted": {
        readonly light: "0 0% 88%";
        readonly dark: "0 0% 60%";
    };
    readonly "--font-sans": {
        readonly light: "\"Geist Sans\", system-ui, sans-serif";
    };
    readonly "--font-mono": {
        readonly light: "\"Geist Mono\", ui-monospace, monospace";
    };
    readonly "--space-1": {
        readonly light: "4px";
    };
    readonly "--space-2": {
        readonly light: "8px";
    };
    readonly "--space-3": {
        readonly light: "12px";
    };
    readonly "--space-4": {
        readonly light: "16px";
    };
    readonly "--radius-sm": {
        readonly light: "4px";
    };
    readonly "--radius-md": {
        readonly light: "8px";
    };
    readonly "--radius-lg": {
        readonly light: "12px";
    };
    readonly "--shadow-sm": {
        readonly light: "0 1px 2px 0 rgb(0 0 0 / 0.05)";
    };
    readonly "--shadow-md": {
        readonly light: "0 4px 6px -1px rgb(0 0 0 / 0.1), 0 2px 4px -2px rgb(0 0 0 / 0.1)";
    };
    readonly "--shadow-lg": {
        readonly light: "0 10px 15px -3px rgb(0 0 0 / 0.1), 0 4px 6px -4px rgb(0 0 0 / 0.1)";
    };
};
//# sourceMappingURL=tokens.d.ts.map