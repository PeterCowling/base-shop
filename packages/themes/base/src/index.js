// packages/themes/base/src/index.ts
export * from "./tokens";
