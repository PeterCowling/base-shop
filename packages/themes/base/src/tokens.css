/* ======================================================================= */
/*  //packages/themes/base/src/tokens.css
    Base-Shop design-system tokens                                         */
/*  Every app should   @import "@themes/base/tokens.css";                  */
/* ======================================================================= */

:root {
  /* semantic colours */
  --color-bg: 0 0% 100%; /* hsl(0 0% 100%)  */
  --color-fg: 0 0% 10%; /* hsl(0 0% 10%)   */
  --color-primary: 220 90% 56%; /* hsl(220 90% 56%)*/
  --color-primary-fg: 0 0% 100%;
  --color-accent: 260 83% 67%;
  --color-muted: 0 0% 88%;

  /* typography */
  --font-sans: "Geist Sans", system-ui, sans-serif;
  --font-mono: "Geist Mono", ui-monospace, monospace;

  /* spacing scale (multiples of 4 px) */
  --space-1: 4px;
  --space-2: 8px;
  --space-3: 12px;
  --space-4: 16px;

  /* radii */
  --radius-sm: 4px;
  --radius-md: 8px;
  --radius-lg: 12px;
}

@media (prefers-color-scheme: dark) {
  :root {
    --color-bg: 0 0% 4%;
    --color-fg: 0 0% 93%;
    --color-primary: 220 90% 66%;
    --color-muted: 0 0% 20%;
  }
}

html.theme-dark {
  --color-bg: 0 0% 4%;
  --color-fg: 0 0% 93%;
  --color-primary: 220 90% 66%;
  --color-muted: 0 0% 20%;
}
