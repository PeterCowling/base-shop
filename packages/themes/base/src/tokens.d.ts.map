{"version":3,"file":"tokens.d.ts","sourceRoot":"","sources":["tokens.ts"],"names":[],"mappings":"AACA;;;GAGG;AAEH,MAAM,WAAW,KAAK;IACpB,yDAAyD;IACzD,QAAQ,CAAC,KAAK,EAAE,MAAM,CAAC;IACvB,yCAAyC;IACzC,QAAQ,CAAC,IAAI,CAAC,EAAE,MAAM,CAAC;CACxB;AAED,yCAAyC;AACzC,MAAM,MAAM,QAAQ,GAAG,MAAM,CAAC,KAAK,MAAM,EAAE,EAAE,KAAK,CAAC,CAAC;AAEpD,eAAO,MAAM,MAAM;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;CAgCU,CAAC"}