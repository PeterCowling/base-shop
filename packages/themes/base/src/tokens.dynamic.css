/* Generated by build-tokens.ts */

:root {
  --color-bg: var(--token-color-bg, 0 0% 100%);
  --color-bg-dark: var(--token-color-bg-dark, 0 0% 4%);
  --color-fg: var(--token-color-fg, 0 0% 10%);
  --color-fg-dark: var(--token-color-fg-dark, 0 0% 93%);
  --color-primary: var(--token-color-primary, 220 90% 56%);
  --color-primary-dark: var(--token-color-primary-dark, 220 90% 66%);
  --color-primary-fg: var(--token-color-primary-fg, 0 0% 100%);
  --color-primary-fg-dark: var(--token-color-primary-fg-dark, 0 0% 10%);
  --color-accent: var(--token-color-accent, 260 83% 70%);
  --color-accent-dark: var(--token-color-accent-dark, 260 83% 70%);
  --color-accent-fg: var(--token-color-accent-fg, 0 0% 10%);
  --color-accent-fg-dark: var(--token-color-accent-fg-dark, 0 0% 10%);
  --color-muted: var(--token-color-muted, 0 0% 88%);
  --color-muted-dark: var(--token-color-muted-dark, 0 0% 60%);
  --font-sans: var(--token-font-sans, "Geist Sans", system-ui, sans-serif);
  --font-mono: var(--token-font-mono, "Geist Mono", ui-monospace, monospace);
  --space-1: var(--token-space-1, 4px);
  --space-2: var(--token-space-2, 8px);
  --space-3: var(--token-space-3, 12px);
  --space-4: var(--token-space-4, 16px);
  --radius-sm: var(--token-radius-sm, 4px);
  --radius-md: var(--token-radius-md, 8px);
  --radius-lg: var(--token-radius-lg, 12px);
  --shadow-sm: var(--token-shadow-sm, 0 1px 2px 0 rgb(0 0 0 / 0.05));
  --shadow-md: var(--token-shadow-md, 0 4px 6px -1px rgb(0 0 0 / 0.1), 0 2px 4px -2px rgb(0 0 0 / 0.1));
  --shadow-lg: var(--token-shadow-lg, 0 10px 15px -3px rgb(0 0 0 / 0.1), 0 4px 6px -4px rgb(0 0 0 / 0.1));
}

@media (prefers-color-scheme: dark) {
  :root {
    --color-bg: var(--color-bg-dark);
    --color-fg: var(--color-fg-dark);
    --color-primary: var(--color-primary-dark);
    --color-primary-fg: var(--color-primary-fg-dark);
    --color-accent: var(--color-accent-dark);
    --color-accent-fg: var(--color-accent-fg-dark);
    --color-muted: var(--color-muted-dark);
  }
}

html.theme-dark {
  --color-bg: var(--color-bg-dark);
  --color-fg: var(--color-fg-dark);
  --color-primary: var(--color-primary-dark);
  --color-primary-fg: var(--color-primary-fg-dark);
  --color-accent: var(--color-accent-dark);
  --color-accent-fg: var(--color-accent-fg-dark);
  --color-muted: var(--color-muted-dark);
}
