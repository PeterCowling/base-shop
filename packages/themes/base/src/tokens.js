// packages/themes/base/tokens.ts
/**
 * Central colour-/spacing-/font-token catalogue.
 * Every token has a `light` value and may provide a `dark` variant.
 */
export const tokens = {
    "--color-bg": { light: "0 0% 100%", dark: "0 0% 4%" },
    "--color-fg": { light: "0 0% 10%", dark: "0 0% 93%" },
    "--color-primary": { light: "220 90% 56%", dark: "220 90% 66%" },
    // Keep foreground on primary light in dark theme for adequate contrast
    "--color-primary-fg": { light: "0 0% 100%", dark: "0 0% 100%" },
    "--color-accent": { light: "260 83% 70%", dark: "260 83% 70%" },
    "--color-accent-fg": { light: "0 0% 10%", dark: "0 0% 10%" },
    "--color-danger": { light: "0 86% 97%", dark: "0 63% 31%" },
    "--color-danger-fg": { light: "0 74% 42%", dark: "0 93% 94%" },
    "--color-success": { light: "142 76% 97%", dark: "142 72% 27%" },
    "--color-success-fg": { light: "142 72% 30%", dark: "142 70% 94%" },
    "--color-warning": { light: "40 90% 96%", dark: "35 90% 30%" },
    "--color-warning-fg": { light: "25 85% 31%", dark: "40 90% 96%" },
    "--color-info": { light: "210 90% 96%", dark: "210 90% 35%" },
    "--color-info-fg": { light: "210 90% 35%", dark: "210 90% 96%" },
    "--color-muted": { light: "0 0% 88%", dark: "0 0% 60%" },
    "--font-sans": { light: 'var(--font-geist-sans)' },
    "--font-mono": { light: 'var(--font-geist-mono)' },
    "--space-1": { light: "4px" },
    "--space-2": { light: "8px" },
    "--space-3": { light: "12px" },
    "--space-4": { light: "16px" },
    "--radius-sm": { light: "4px" },
    "--radius-md": { light: "8px" },
    "--radius-lg": { light: "12px" },
    "--shadow-sm": { light: "0 1px 2px 0 rgb(0 0 0 / 0.05)" },
    "--shadow-md": {
        light: "0 4px 6px -1px rgb(0 0 0 / 0.1), 0 2px 4px -2px rgb(0 0 0 / 0.1)",
    },
    "--shadow-lg": {
        light: "0 10px 15px -3px rgb(0 0 0 / 0.1), 0 4px 6px -4px rgb(0 0 0 / 0.1)",
    },
};
