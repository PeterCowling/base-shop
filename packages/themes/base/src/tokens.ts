// packages/themes/base/tokens.ts
/**
 * Central colour-/spacing-/font-token catalogue.
 * Every token has a `light` value and may provide a `dark` variant.
 */

export interface Token {
  /** CSS colour / length / font list used in light mode */
  readonly light: string;
  /** Optional counterpart for dark mode */
  readonly dark?: string;
}

/** `--token-name`: `{ light, dark? }` */
export type TokenMap = Record<`--${string}`, Token>;

export const tokens = {
  // Neutral core
  "--color-bg": { light: "0 0% 100%", dark: "0 0% 4%" },
  "--color-fg": { light: "0 0% 10%", dark: "0 0% 93%" },
  // Neutral extensions (12-step semantic mapping)
  "--color-bg-1": { light: "0 0% 100%", dark: "0 0% 4%" },
  "--color-bg-2": { light: "0 0% 98%", dark: "0 0% 6%" },
  "--color-bg-3": { light: "0 0% 96%", dark: "0 0% 9%" },
  "--color-bg-4": { light: "0 0% 94%", dark: "0 0% 12%" },
  "--color-bg-5": { light: "0 0% 92%", dark: "0 0% 15%" },
  "--color-panel": { light: "0 0% 96%", dark: "0 0% 12%" },
  "--color-inset": { light: "0 0% 98%", dark: "0 0% 9%" },
  "--color-border": { light: "0 0% 80%", dark: "0 0% 30%" },
  "--color-border-strong": { light: "0 0% 65%", dark: "0 0% 45%" },
  "--color-border-muted": { light: "0 0% 88%", dark: "0 0% 22%" },
  "--color-fg-muted": { light: "0 0% 40%", dark: "0 0% 75%" },
  // Brand families
  "--color-primary": { light: "220 90% 56%", dark: "220 90% 66%" },
  // In dark theme, foreground on primary should remain light for contrast
  "--color-primary-fg": { light: "0 0% 100%", dark: "0 0% 100%" },
  "--color-primary-soft": { light: "220 90% 96%", dark: "220 90% 18%" },
  "--color-primary-hover": { light: "220 90% 50%", dark: "220 90% 72%" },
  "--color-primary-active": { light: "220 90% 45%", dark: "220 90% 78%" },
  "--color-accent": { light: "260 83% 70%", dark: "260 83% 70%" },
  "--color-accent-fg": { light: "0 0% 10%", dark: "0 0% 10%" },
  "--color-accent-soft": { light: "260 83% 97%", dark: "260 83% 20%" },
  "--color-danger": { light: "0 86% 97%", dark: "0 63% 31%" },
  "--color-danger-fg": { light: "0 74% 42%", dark: "0 93% 94%" },
  "--color-danger-soft": { light: "0 100% 98%", dark: "0 50% 20%" },
  "--color-success": { light: "142 76% 97%", dark: "142 72% 27%" },
  "--color-success-fg": { light: "142 72% 30%", dark: "142 70% 94%" },
  "--color-success-soft": { light: "142 76% 96%", dark: "142 60% 18%" },
  "--color-warning": { light: "40 90% 96%", dark: "35 90% 30%" },
  "--color-warning-fg": { light: "25 85% 31%", dark: "40 90% 96%" },
  "--color-warning-soft": { light: "40 90% 95%", dark: "35 70% 20%" },
  "--color-info": { light: "210 90% 96%", dark: "210 90% 35%" },
  "--color-info-fg": { light: "210 90% 35%", dark: "210 90% 96%" },
  "--color-info-soft": { light: "210 90% 95%", dark: "210 70% 20%" },
  "--color-muted": { light: "0 0% 88%", dark: "0 0% 60%" },
  // Accessible link color (AA on light surfaces)
  "--color-link": { light: "220 75% 40%", dark: "220 80% 70%" },
  // Interaction helpers
  "--color-focus-ring": { light: "220 90% 56%", dark: "220 90% 66%" },
  "--color-selection": { light: "260 83% 92%", dark: "220 30% 40%" },
  "--color-highlight": { light: "260 83% 97%", dark: "220 25% 30%" },
  "--color-muted-fg": { light: "0 0% 20%", dark: "0 0% 92%" },
  "--color-muted-border": { light: "0 0% 72%", dark: "0 0% 40%" },
  // Translucent overlays
  "--overlay-scrim-1": { light: "0 0% 0% / 0.40", dark: "0 0% 100% / 0.40" },
  "--overlay-scrim-2": { light: "0 0% 0% / 0.64", dark: "0 0% 100% / 0.64" },
  // Foreground color suitable for hero backgrounds (paired with bg-hero-contrast)
  "--hero-fg": { light: "0 0% 100%", dark: "0 0% 100%" },
  "--font-sans": { light: 'var(--font-geist-sans)' },
  "--font-mono": { light: 'var(--font-geist-mono)' },
  // Three-font typography model
  // Body copy defaults to sans until configured
  "--font-body": { light: 'var(--font-sans)' },
  // Primary and secondary heading families default to body until configured
  "--font-heading-1": { light: 'var(--font-sans)' },
  "--font-heading-2": { light: 'var(--font-sans)' },
  // Text theme hooks used by the CMS page builder to extract typography themes
  "--typography-body-font-family": { light: 'var(--font-body)' },
  "--text-heading-1-font-family": { light: 'var(--font-heading-1)' },
  "--text-heading-2-font-family": { light: 'var(--font-heading-2)' },
  "--space-1": { light: "4px" },
  "--space-2": { light: "8px" },
  "--space-3": { light: "12px" },
  "--space-4": { light: "16px" },
  "--radius-sm": { light: "4px" },
  "--radius-md": { light: "8px" },
  "--radius-lg": { light: "12px" },
  "--shadow-sm": { light: "0 1px 2px 0 rgb(0 0 0 / 0.05)" },
  "--shadow-md": {
    light: "0 4px 6px -1px rgb(0 0 0 / 0.1), 0 2px 4px -2px rgb(0 0 0 / 0.1)",
  },
  "--shadow-lg": {
    light: "0 10px 15px -3px rgb(0 0 0 / 0.1), 0 4px 6px -4px rgb(0 0 0 / 0.1)",
  },
  // Gradient tokens (used by bg-hero utility)
  "--gradient-hero-from": { light: "234 89% 60%" },
  "--gradient-hero-via": { light: "270 83% 60%" },
  "--gradient-hero-to": { light: "222 47% 11%" },
  // Overlay used by bg-hero-contrast to guarantee text contrast
  "--hero-contrast-overlay": { light: "0 0% 0% / 0.55", dark: "0 0% 0% / 0.55" },
} as const satisfies TokenMap;
