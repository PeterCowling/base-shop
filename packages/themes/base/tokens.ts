// packages/themes/base/tokens.ts
/**
 * Central colour-/spacing-/font-token catalogue.
 * Every token has a `light` value and may provide a `dark` variant.
 */

export interface Token {
  /** CSS colour / length / font list used in light mode */
  readonly light: string;
  /** Optional counterpart for dark mode */
  readonly dark?: string;
}

/** `--token-name`: `{ light, dark? }` */
export type TokenMap = Record<`--${string}`, Token>;

/* eslint-disable prettier/prettier */
export const tokens = {
  "--color-bg": { light: "0 0% 100%", dark: "0 0% 4%" },
  "--color-fg": { light: "0 0% 10%", dark: "0 0% 93%" },
  "--color-primary": { light: "220 90% 56%", dark: "220 90% 66%" },
  "--color-primary-fg": { light: "0 0% 100%" },
  "--color-accent": { light: "260 83% 67%" },
  "--color-muted": { light: "0 0% 88%", dark: "0 0% 20%" },
  "--font-sans": { light: '"Geist Sans", system-ui, sans-serif' },
  "--font-mono": { light: '"Geist Mono", ui-monospace, monospace' },
  "--space-1": { light: "4px" },
  "--space-2": { light: "8px" },
  "--space-3": { light: "12px" },
  "--space-4": { light: "16px" },
  "--radius-sm": { light: "4px" },
  "--radius-md": { light: "8px" },
  "--radius-lg": { light: "12px" },
} as const satisfies TokenMap;
