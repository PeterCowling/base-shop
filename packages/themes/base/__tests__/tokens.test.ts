import fs from 'node:fs';
import path from 'node:path';
import { tokens } from '../src/tokens';

const expectedKeys = [
  '--color-bg',
  '--color-fg',
  '--color-primary',
  '--color-primary-fg',
  '--color-accent',
  '--color-accent-fg',
  '--color-danger',
  '--color-danger-fg',
  '--color-success',
  '--color-success-fg',
  '--color-warning',
  '--color-warning-fg',
  '--color-info',
  '--color-info-fg',
  '--color-muted',
  '--font-sans',
  '--font-mono',
  '--space-1',
  '--space-2',
  '--space-3',
  '--space-4',
  '--radius-sm',
  '--radius-md',
  '--radius-lg',
  '--shadow-sm',
  '--shadow-md',
  '--shadow-lg',
] as const;

describe('base theme tokens', () => {
  test('exports expected token keys', () => {
    expect(Object.keys(tokens).sort()).toEqual([...expectedKeys].sort());
  });

  test('css contains token variables', () => {
    const cssDir = path.join(__dirname, '..', 'src');
    const cssFiles = ['tokens.css', 'tokens.dynamic.css']
      .map((f) => path.join(cssDir, f))
      .filter(fs.existsSync);
    const css = cssFiles.map((f) => fs.readFileSync(f, 'utf8')).join('\n');

    for (const key of expectedKeys) {
      expect(css).toContain(key);
    }
  });

  test('tokens snapshot', () => {
    expect(tokens).toMatchSnapshot();
  });
});
