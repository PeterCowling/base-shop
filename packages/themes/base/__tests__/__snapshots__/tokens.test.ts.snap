// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`base theme tokens tokens snapshot 1`] = `
{
  "--color-accent": {
    "dark": "260 83% 70%",
    "light": "260 83% 70%",
  },
  "--color-accent-fg": {
    "dark": "0 0% 10%",
    "light": "0 0% 10%",
  },
  "--color-bg": {
    "dark": "0 0% 4%",
    "light": "0 0% 100%",
  },
  "--color-fg": {
    "dark": "0 0% 93%",
    "light": "0 0% 10%",
  },
  "--color-muted": {
    "dark": "0 0% 60%",
    "light": "0 0% 88%",
  },
  "--color-primary": {
    "dark": "220 90% 66%",
    "light": "220 90% 56%",
  },
  "--color-primary-fg": {
    "dark": "0 0% 10%",
    "light": "0 0% 100%",
  },
  "--font-mono": {
    "light": ""Geist Mono", ui-monospace, monospace",
  },
  "--font-sans": {
    "light": ""Geist Sans", system-ui, sans-serif",
  },
  "--radius-lg": {
    "light": "12px",
  },
  "--radius-md": {
    "light": "8px",
  },
  "--radius-sm": {
    "light": "4px",
  },
  "--shadow-lg": {
    "light": "0 10px 15px -3px rgb(0 0 0 / 0.1), 0 4px 6px -4px rgb(0 0 0 / 0.1)",
  },
  "--shadow-md": {
    "light": "0 4px 6px -1px rgb(0 0 0 / 0.1), 0 2px 4px -2px rgb(0 0 0 / 0.1)",
  },
  "--shadow-sm": {
    "light": "0 1px 2px 0 rgb(0 0 0 / 0.05)",
  },
  "--space-1": {
    "light": "4px",
  },
  "--space-2": {
    "light": "8px",
  },
  "--space-3": {
    "light": "12px",
  },
  "--space-4": {
    "light": "16px",
  },
}
`;
